seed/seedingScript.js

```diff
@@ -18,8 +18,19 @@ try {
   const mainImagesStream = fs.createWriteStream(filePath + '/mainImages.csv');
   generateCSV(mainImagesStream, createMainImageRecords, 1e7, () => {
     mainImagesStream.end();
-    console.log('Successfully wrote mainImage records to CSV')
+    console.log('Successfully wrote mainImage records to CSV');
   });
 } catch (err) {
   if (err) {console.log(err)}
 }
+
+try {
+  const couchStream = fs.createWriteStream(filePath + '/couch.csv');
+  couchStream.write('_id\theaderImage\tmainImages\n');
```

Comment:
oh ok so is the \t a short hand way of separating csv values? i've had success with just using a comma, but could begin to imagine how the \t might be more dynamic and interesting

Reply:
yup, exactly. you can separate your csv with whatever value you want, ie commas, tabs, spaces, etc. technically you could separate it with a string like `'separator'` and then you have to tell whatever import tool how youre separating columns.